### DISCUSSION TITLE
Delete the grid specific to neutral items

### DISCUSSION THREAD

**Discussion by ReadAaagain** [ValveSoftware/Dota2-Gameplay#2211] [Ideas]
Delete the grid specific to neutral items so that they can be placed in the normal grid, and use the right mouse button to activate a neutral item and set it as in use, while other neutral items will not be used, even if they are placed in the normal grid.

**Comment by kortana45** [comment]
As i know neutral items can now be placed in the normal grid alongside other items. To activate a neutral item, simply right-click on it and set it as "in use." Other neutral items that are placed in the normal grid will not be activated or used.                                      [TargetPayandBenefits Login](https://www.targetpayandbenefits.review/)

**Comment by Markiye** [comment]
+1 to this idea[.](https://megamodapk.com/nulls-brawl-apk/)
